gpg-mailgate.py

```diff
@@ -469,7 +469,7 @@ def encrypt_all_payloads_mime( message, gpg_to_cmdline ):
 
 def encrypt_payload( payload, gpg_to_cmdline, check_nested = True ):
 
-	raw_payload = payload.get_payload(decode=True)
+	raw_payload = payload.get_payload().encode()
```

Comment:
why remove the decode here? Sounds like this is required to support mime based encodings like base64 encoded messages, ect..